tutorials/migrating-rails-app-from-heroku-to-google-appengine.md

```diff
@@ -255,43 +251,40 @@ them as environment variables into your app's runtime environment.
 App Engine provides two ways to set environment variables, depending on how
 you'd like to manage and secure the content. First, you can set them directly
 in the `app.yaml` file by creating an `env_variables` section, which App Engine
-will inject into your app's runtime environment. For example:
+injects into your app's runtime environment. For example:
 
     env_variables:
       DATABASE_USER: postgres
       RAILS_MAX_THREADS: "5"
 
-This is useful for simple configuration. However, you will typically want to
+This is useful for simple configuration. However, you typically want to
 save your `app.yaml` config file to source control, so it is not a good place
 for secrets such as your database password or your `SECRET_KEY_BASE`. This
 kind of sensitive data can instead be managed securely using the
 [Runtime Configuration Service](https://cloud.google.com/deployment-manager/runtime-configurator/),
 a Google service similar to the Heroku config system.
 
-In your app, you probably need to manage at least two secret values: the
+In your app, you likely need to manage at least two secret values: the
 `DATABASE_PASSWORD`, and the `SECRET_KEY_BASE`. Let's store them in Runtime
-Configuration.
-
-1.  Enable the Runtime Config API in the Cloud Console
-    [on this page](https://console.cloud.google.com/apis/library/runtimeconfig.googleapis.com/).
+Configuration:
 
-2.  Create a configuration called `my-env` as follows:
+1.  Create a configuration named `my-env` by running the following command:
 
         gcloud beta runtime-config configs create my-env
 
-3.  View the config values for your Heroku app.
+1.  View the config values for your Heroku app:
 
         heroku config
 
     Copy the `SECRET_KEY_BASE` value so you can paste it in the next command.
 
-4.  Set the `SECRET_KEY_BASE` in your Google Runtime Config.
+1.  Set the `SECRET_KEY_BASE` in your Google Runtime Config.
 
         gcloud beta runtime-config configs variables set \
           --config-name=my-env --is-text \
           SECRET_KEY_BASE "VeryLongStringCopiedFromHeroku"
 
-5.  Previously you set up your `database.yml` configuration to read from the
+1.  Previously you set up your `database.yml` configuration to read from the
     `DATABASE_PASSWORD` environment variable. Set that up:
 
         gcloud beta runtime-config configs variables set \
```

Comment:
Maybe we should use `[MY_DATABASE_PASSWORD]` here for consistency with how we're expressing substitutions elsewhere?

Reply:
Oops, thanks for catching. Done.